### ISSUE TITLE
[Request] Placeholder Updates/Catch-up

### ISSUE THREAD

**Issue by mercurialmusic** [Zrips/CMI#8167]
Not a bug, but perhaps forgotten: the newer options for ride and the pvp/pve toggles don't seem to have the equivalent %cmi_user_toggle_[feature]% support yet, which would be greatly appreciated!

**Comment by CloudeLecaw** [comment]
you landet in the wrong area :-)

create your request here
https://github.com/Zrips/CMI/discussions/new?category=enhancement

![Screenshot_2023-10-13-06-40-47-850_com android chrome-edit](https://github.com/Zrips/CMI/assets/12080017/eeaa9804-3431-4cac-a52a-d422ba47dadb)

**Comment by mercurialmusic** [comment]
> you landet in the wrong area :-)
> 
> create your request here https://github.com/Zrips/CMI/discussions/new?category=enhancement

Yup, I'm aware, but this seemed more like an accidental omission/oversight rather than an enhancement request. There seems to be some sort of internal placeholder in place already since the /cmi options menu can determine the current status, but for those of us who implement our own options menu for whatever reason, those are not available to us, which may indeed be a bug if internal placeholders are working but cannot be parsed through the placeholders available to admins.

**Comment by CloudeLecaw** [comment]
what about these?
![Uploading Screenshot_2023-10-13-10-22-49-620_com.android.chrome-edit.jpg…]()

**Comment by CloudeLecaw** [comment]
![Screenshot_2023-10-13-10-22-49-620_com android chrome-edit](https://github.com/Zrips/CMI/assets/12080017/f5c28ff5-2452-4156-ae06-6014aa839ef7)
